workflows/Linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
```

Comment:
- [ ] [OPTIONAL] Do you need this extra `linters.yml` here. I think the one in the `.github/workflows` folder is enough.